### PR TITLE
add clicknium recorder docs

### PR DESCRIPTION
**Pull request by DiceyClickcorp**

### CODE REVIEW

File: doc/developtools/recorder/validate.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+2. Press the button `Validate`, popup window as follows:  
+![validate variable window](../../img/validate_variable_window.png)
+3. Input varible value, then press the button `Continue`
+4. Get validation result refer to [Validation result](#validation-result)
+
+### Single locator with mutiple windows
+1. If your locator can be found in mutiple windows, press the button `Validate`, popup window will as follows:  
+![validate muti windows](../../img/validate_muti_window.png)
+2. Select the index of window to get correct window, then press the button `Continue`
+3. Get validation result refer to [Validation result](#validation-result)
```
Comment (Luluyu2022) [inline doc/developtools/recorder/validate.md:35]: referring

File: doc/developtools/recorder/validate.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+### Mutiple similar locators
+1. Record a locator, and modify the locator with wildword as follows:
+![locator wildword](../../img/locator_wildword.png)
+2. Select the index of the element, then Press `OK`
```
Comment (Luluyu2022) [inline doc/developtools/recorder/validate.md:40]: and click button "ok"

File: doc/developtools/recorder/validate.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+Press the button `Validate`, and get validation result refer to [Validation result](#validation-result).
+
+### Single locator with variables
+
+1. Record a locator, and modify the locator with variables, for more about variables, please refer to [Parametric Locator](./doc/automation/parametric_locator.md).
+![locator variables](../../img/locator_variables.png)
+2. Press the button `Validate`, popup window as follows:  
+![validate variable window](../../img/validate_variable_window.png)
+3. Input varible value, then press the button `Continue`
```
Comment (Luluyu2022) [inline doc/developtools/recorder/validate.md:28]: Input the value of the variable, and click the button `Continue`

File: doc/developtools/recorder/validate.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+- [Validation result](#validation-result)
+
+## Start validate
+You can validate from locator viewer tab in Visual Studio Code. 
+    ![vscode validate](../../img/recorder_validate_vscode.png)
+
+## Locator validation
+### Single locator
+
+Press the button `Validate`, and get validation result refer to [Validation result](#validation-result).
```
Comment (Luluyu2022) [inline doc/developtools/recorder/validate.md:20]: Click the button `Validate` and get the validation result referring to [Validation result](#validation-result).

File: doc/developtools/recorder/validate.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+- [Start validate](#start-validate)
+- [Locator validation](#locator-validation)
+    - [Single locator](#single-locator)
+    - [Single locator with variables](#single-locator-with-variables)
+    - [Single locator with mutiple windows](#single-locator-with-mutiple-windows)
+    - [Mutiple similar locators](#mutiple-similar-locators)
+- [Validation result](#validation-result)
+
+## Start validate
+You can validate from locator viewer tab in Visual Studio Code.
```
Comment (Luluyu2022) [inline doc/developtools/recorder/validate.md:14]: You can start "validate" from the locator viewer tab in Visual Studio Code.

File: doc/developtools/recorder/record.md [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+![record complete](../../img/recorder_complete.png)
+
+### Continuous record
+
+1. Operation multiple times "Select UI element and Press `Ctrl` + click" to add more locators
+2. Press button `Complete` on recorder panel
+
+### Image record
+
+1. Select UI element
```
Comment (Luluyu2022) [inline doc/developtools/recorder/record.md:45]: Select a UI element

**Review by Luluyu2022 — COMMENTED** [review]
grammar update.
capitalize every word in a phrase or sentence
"click the button" is better.

**Review by robinbob — APPROVED** [review]

**Pull request merged**